miio/yeelight_dual_switch.py

```diff
@@ -0,0 +1,233 @@
+import enum
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .exceptions import DeviceException
+from .miot_device import MiotDevice
+
+
+class YeelightDualControlModuleException(DeviceException):
+    pass
+
+
+class Mode(enum.Enum):
+    Off = 0
+    On = 1
+
+
+_MAPPING = {
+    # http://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:switch:0000A003:yeelink-sw1:1:0000C809
+    # First Switch (siid=2)
+    "switch_1_status": {"siid": 2, "piid": 1},  # bool
+    "switch_1_default_state": {"siid": 2, "piid": 2},  # 0 - Off, 1 - On
+    "switch_1_delay": {"siid": 2, "piid": 3},  # -1 - Off, [1, 43200] - delay in sec
+    # Second Switch (siid=3)
+    "switch_2_status": {"siid": 3, "piid": 1},  # bool
+    "switch_2_default_state": {"siid": 3, "piid": 2},  # 0 - Off, 1 - On
+    "switch_2_delay": {"siid": 3, "piid": 3},  # -1 - Off, [1, 43200] - delay in sec
+    # Extensions (siid=4)
+    "interlock": {"siid": 4, "piid": 1},  # bool
+    "flex_mode": {"siid": 4, "piid": 2},  # 0 - Off, 1 - On
+    "rc_list": {"siid": 4, "piid": 3},  # string
+    "rc_list_for_del": {"siid": 4, "piid": 4},  # string
+    "toggle": {"siid": 4, "piid": 5},  # 0 - channel0, 1 - channel1
+}
+
+
+class DualControlModuleStatus:
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
```

Comment:
Please add a docstring showing an example response from the device, see other miot devices for examples.